src/rfac/rfmklist.c

```
@@ -0,0 +1,169 @@
+    fprintf(STDCTR, "(cr_mklist): (the) n_eng: %d\n", n_eng);
+    #endif
+
+  } /* Higher density of energy points in theor. list */
+ 
+  #ifdef WRITE
+  str_e_int = fopen("rfmklist.exp", "w");
+  str_t_int = fopen("rfmklist.the", "w");
+
+  for(i_eng = 0; i_eng < n_eng; i_eng++)
+  {
+    fprintf(str_e_int, "%f %e\n", eng[i_eng], e_int[i_eng]/e_max);
```

Cppcheck (2.19.0)

[warning] If resource allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfResources)